README.md

```diff
@@ -23,23 +23,23 @@ it accordingly if you want run in a production env.
 You need to create the docker container:
 
 ```sh
-$ docker-compose up -d
+$ docker compose up -d
```

Comment:
Use [Compose V2](https://docs.docker.com/compose/), instead of `docker-compose`

Reply:
Acho de boa a substituição. Mas como ainda é meio recente, acho melhor deixar uma nota que substituir `docker compose` por `docker-compose` funcionaria normal (talvez linkando [esse tópico](https://docs.docker.com/compose/cli-command-compatibility/)). O que acha?